dotnet/src/SemanticKernel.Abstractions/AI/TextCompletion/TextCompletionExtensions.cs

```diff
@@ -48,7 +48,7 @@ public static class TextCompletionExtensions
     {
         var completionResults = textCompletion.GetStreamingCompletionsAsync(text, requestSettings, cancellationToken);
 
-        await foreach (var completionResult in completionResults)
+        await foreach (var completionResult in completionResults.WithCancellation(cancellationToken))
```

Comment:
Not needed -- the implementations use [EnumeratorCancellation] attibute, which does the same thing. Perhaps we should squash this resharper rule.

Reply:
Reverted in [that commit](https://github.com/microsoft/semantic-kernel/pull/1914/commits/07d24c03677da3ebbc1df463f7612a95835c7e4d)